convene-web/app/models/invitation.rb

```diff
@@ -0,0 +1,2 @@
+class Invitation < ApplicationRecord
+end
```

Comment:
Not blocking:

We may want to communicate what the "Job" of the invitation is by adding some comments.

Things that are useful in this kind of documentation are:

1. What other parts of the system does an Invitation interact with and why?
2. What Feature files are particularly relevant for this model?


Here's an example, written in [YARDoc syntax](https://yardoc.org/)

```
# An {Invitation} offers a {Person} the opportunity to join a {Space}. When an {Invitation} is accepted, it creates a {SpaceMembership}. 
# For more information, review the {file:features/spaces/invitation.feature Space Invitation Feature Definition.}